README.md

```diff
@@ -2,9 +2,11 @@
 
 [![PyPI version](https://badge.fury.io/py/vertica-python.png)](http://badge.fury.io/py/vertica-python)
 
-0.4.2 breaks some of the older query interfaces (row_handler callback, and connection.query).
+0.4.x breaks some of the older query interfaces (row_handler callback, and connection.query).
 It replaces the row_handler callback with an iterate() method. Please see examples below
 
+If you are on 0.4.1 - 0.4.3, please upgrade to 0.4.5 as there are various bug fixes
+
 vertica-python is a native Python adapter for the Vertica (http://www.vertica.com) database.
 
 vertica-python is currently in alpha stage; it has been tested for functionality, but does not have a test suite. Please use with caution, and feel free to submit issues and/or pull requests.
```

Comment:
I think this is at least a beta by now. You may also want to add some notes about running the unit tests before submitting a PR.